### DISCUSSION TITLE
Building rkg68 for via

### DISCUSSION THREAD

**Discussion by resuna** [community/community#130507] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was trying to build rkg68:via and quantum/dynamic_keymap.c depends on some VIAL macros, but when you enable VIAL in via/rules.mk it doesn't complain about them any more but it now needs vial.json. Please advise. This is my first keyboard I am trying to build QMK for and I am but an egg for this codebase, so I am sure that if I try and make a vial.json from scratch I will make a mess of it.

If there is another one I could use as a model I think I can manage, qmk is not my first rodeo, I have many repos.

**Comment by ebndev** [comment]
Hi @resuna, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
